paddle/operators/pool_op.cc

```diff
@@ -80,34 +80,31 @@ Pool2dOpMaker::Pool2dOpMaker(framework::OpProto *proto,
             "the number of channels, H and W is the height and "
             "width of feature.");
 
-  AddAttr<std::string>("pooling_type",
-                       "Pooling_type of pooling operator."
+  AddAttr<std::string>("poolingType",
```

Comment:
Google C++ style did not specify the constant string naming rules, but since this attribute is going to be used when user define operators using python, following python style seems resonable?